test/functional.js

```diff
@@ -377,11 +421,13 @@ describe('Functional tests using webpack', function() {
             testSetup.runWebpack(config, (webpackAssert) => {
                 expect(config.outputPath).to.be.a.directory()
                     .with.files([
-                        '0.d002be21.js', // chunks are also versioned
-                        'main.292c0347.js',
-                        'h1.c84caea6.css',
-                        'bg.483832e4.css',
-                        'manifest.json'
+                        '0.8256b1ad.js', // chunks are also versioned
+                        '0.8ec31654.css',
+                        'main.ba427376.js',
+                        'h1.8ec31654.css',
+                        'bg.0ec2735b.css',
+                        'manifest.json',
+                        'entrypoints.json'
```

Comment:
I suggest asserting the content of `entrypoints.json` too in this test using versionning